cypress/e2e/pages/create_wallet.pages.js

```diff
@@ -41,7 +40,7 @@ export const defaultSepoliaPlaceholder = 'Sepolia Safe'
 const welcomeToSafeStr = 'Welcome to Safe'
 
 export function verifyNewSafeDialogModal() {
-  main.verifyElementsIsVisible([safeBackupAlert, dialogConfirmBtn])
+  main.verifyElementsIsVisible([dialogConfirmBtn])
```

Comment:
@mike10ca I have to adjust this line in order for the test to work again since the backup option for counterfactual safes is removed in this PR.